### PR TITLE
document: some minor fix/update

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
1. JVMChaos require Linux kernel 4.1 or above
2. update wrong Chaos Mesh code download URL
3. chaosd required glibc 2.17 or above, instead of Linux kernel

### CODE REVIEW

File: docs/simulate-jvm-application-chaos.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ Chaos Mesh simulates the faults of JVM application through [Byteman](https://git
 
 This document describes how to use Chaos Mesh to create the above fault types of JVM experiments.
 
+::: note
```
Comment (en-jin19) [inline docs/simulate-jvm-application-chaos.md:16]:
```suggestion
:::note
```

File: docs/simulate-jvm-application-chaos.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ Chaos Mesh simulates the faults of JVM application through [Byteman](https://git
...
+
+Make sure your Linux kernel must be v4.1 or later.
```
Comment (en-jin19) [inline docs/simulate-jvm-application-chaos.md:18]:
```suggestion
Make sure your Linux kernel is v4.1 or a later version.
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-jvm-application-chaos.md [hunk 3]
```diff
@@ -13,6 +13,12 @@ Chaos Mesh 通过 [Byteman](https://github.com/chaos-mesh/byteman) 模拟 JVM 
 
 本文主要介绍如何创建以上故障类型的 JVM 实验。
 
+::: note
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-jvm-application-chaos.md:16]:
```suggestion
:::note注意
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/simulate-jvm-application-chaos.md [hunk 4]
```diff
@@ -13,6 +13,12 @@ Chaos Mesh 通过 [Byteman](https://github.com/chaos-mesh/byteman) 模拟 JVM 
 
 本文主要介绍如何创建以上故障类型的 JVM 实验。
 
+::: note
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.1/simulate-jvm-application-chaos.md:16]:
```suggestion
:::note 注意
```

File: blog/2021-09-15-run-chaos-experiments-on-physical-machines.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ Without further ado, let’s give it a try.
 
 ## How to use chaosd
 
-In this section, I will walk you through how to inject a network fault with chaosd. Your Linux kernel version must be v2.17 or later.
+In this section, I will walk you through how to inject a network fault with chaosd. Your glibc version must be v2.17 or later.
```
Comment (en-jin19) [inline blog/2021-09-15-run-chaos-experiments-on-physical-machines.md:34]:
```suggestion
In this section, I will walk you through how to inject a network fault with chaosd. Your glibc version must be v2.17 or later versions.
```

File: docs/chaosd-overview.md [hunk 6]
```diff
@@ -26,7 +26,7 @@ For details about the introduction and usage of each fault type, refer to the re
 
 ### Operating environment
 
-Your Linux kernel version must be v2.17 or later.
+Your glibc version must be v2.17 or later.
```
Comment (en-jin19) [inline docs/chaosd-overview.md:29]:
```suggestion
Your glibc version must be v2.17 or later versions.
```

File: docs/simulate-jvm-application-chaos.md (hunk 2 above)
Comment (en-jin19) [inline docs/simulate-jvm-application-chaos.md:18]:
```suggestion
Your Linux kernel must be v4.1 or later versions.
```

File: versioned_docs/version-2.1.1/simulate-jvm-application-chaos.md [hunk 7]
```diff
@@ -13,6 +13,12 @@ Chaos Mesh simulates the faults of JVM application through [Byteman](https://git
 
 This document describes how to use Chaos Mesh to create the above fault types of JVM experiments.
 
+::: note
```
Comment (en-jin19) [inline versioned_docs/version-2.1.1/simulate-jvm-application-chaos.md:16]:
```suggestion
:::note
```

File: versioned_docs/version-2.1.1/simulate-jvm-application-chaos.md [hunk 8]
```diff
@@ -13,6 +13,12 @@ Chaos Mesh simulates the faults of JVM application through [Byteman](https://git
...
+
+Make sure your Linux kernel must be v4.1 or later.
```
Comment (en-jin19) [inline versioned_docs/version-2.1.1/simulate-jvm-application-chaos.md:18]:
```suggestion
Your Linux kernel must be v4.1 or later versions.
```

File: docs/simulate-jvm-application-chaos.md (hunk 2 above)
Comment (WangXiangUSTC) [reply docs/simulate-jvm-application-chaos.md:18]: 需要在前面加上 `Make sure` 吗

**Review by cwen0 — APPROVED** [review]
LGTM

File: versioned_docs/version-2.1.2/simulate-jvm-application-chaos.md [hunk 9]
```diff
@@ -13,6 +13,12 @@ Chaos Mesh simulates the faults of JVM application through [Byteman](https://git
 
 This document describes how to use Chaos Mesh to create the above fault types of JVM experiments.
 
+:::note
+
+Make sure your Linux kernel must be v4.1 or later.
```
Comment (en-jin19) [inline versioned_docs/version-2.1.2/simulate-jvm-application-chaos.md:18]:
```suggestion
Your Linux kernel must be v4.1 or later.
```

**Review by en-jin19 — APPROVED** [review]
rest LGTM

**Pull request merged**